kokoro/gcp_ubuntu/cmake/build.sh

```diff
@@ -36,5 +36,8 @@ cd ${KOKORO_ARTIFACTS_DIR?}/github/iree
 echo "Initializing submodules"
 ./scripts/git/submodule_versions.py init
 
-echo "Building and testing with cmake"
+echo "Building with cmake"
 ./build_tools/cmake/build.sh
```

Comment:
With Bazel, a single invocation can fail to build some targets but still run tests for targets that were able to build successfully. Can we get similar behavior here, or are we stuck with the script completing early if there is any build failure? I'd like to avoid needing to re-run the entire script / presubmit after fixing a build issue just to see test failures.

Reply:
I have no idea how to do that with CMake. I wouldn't say it's not possible, but I'd prefer to merge this as-is, since currently you can't see test failures at all :-D

Reply:
Fair enough :P

Maybe add a TODO / comment?